topics/persistence.md

```diff
@@ -203,6 +222,25 @@ corruption happened to be in the initial part of the file.
 Log rewriting uses the same copy-on-write trick already in use for
 snapshotting.  This is how it works:
 
+**Redis >= 7.0**
+
+* Redis [forks](http://linux.die.net/man/2/fork), so now we have a child
+and a parent process.
+
+* The child starts writing the new AOF in a temporary file.
+
+* The parent opens a new increments AOF file to continue writing updates.
+  If the rewriting fails, the old base and increments files (if we have) plus this newly opened increments file represent the complete updated dataset,
+  so we are safe.
+  
+* When the child is done rewriting the base file, the parent gets a signal,
+and uses the newly opened increment file and child generated base file to build an temp manifest,
+and persist it.
```

Comment:
maybe it would be clearer if we point out that there are no usually a base file and two incremental files (+manifest).

Reply:
At this stage there's normally one new inc file, one old inc file, one new base file and one old base file. When done writing the new base file we just use it and the **new** inc file. This is what it says here. What do you think is missing?

Reply:
before AOFRW, we may have: `BASE1 + INCR1 + MANIFEST`
start AOFRW, we have:  `BASE1 + INCR1 + INCR2 + MANIFEST`
child process exit normally AOFRW (in doneHandler), we have:  `BASE1 + INCR1 + INCR2 + MANIFEST +  BASE2 `
AOFRW finish (del history), we have: `BASE2 + INCR2`